src/components/avatar.js

```diff
@@ -0,0 +1,31 @@
+(() => ({
+  name: 'Avatar',
+  type: 'CONTENT_COMPONENT',
+  allowedTypes: ['CONTENT_COMPONENT'],
+  orientation: 'HORIZONTAL',
+  jsx: (() => {
+    const { Avatar } = window.MaterialUI.Core;
+    const { env } = B;
+    const isDev = env === 'dev';
+    const { imgUrl, imgAlt, variant } = options;
+
+    const AvatarComponent = (
+      <Avatar
+        variant={variant}
+        alt={imgAlt}
+        src={imgUrl}
+        className={classes.avatar}
+      >
+        {children}
+      </Avatar>
+    );
+
+    return isDev ? <div>{AvatarComponent}</div> : AvatarComponent;
```

Comment:
Missing a placeholder

Reply:
<img width="59" alt="Schermafbeelding 2020-05-22 om 13 05 37" src="https://user-images.githubusercontent.com/6944/82661782-f4561e00-9c2c-11ea-9187-ba7551dbbad1.png">

This looks pretty placeholdery to me? What kind of placeholder do you expect?

Reply:
Same as the image component